src/main/java/ch/bfh/bti7081/s2018/green/presenters/HeaderPresenter.java

```diff
@@ -41,8 +41,8 @@ private void enteredView() {
 				data.setCurrentPatient(event.getValue());
 
 				// Upon switch to other patient: Go to Journal-View
-				if(data.getCurrentNavigator() != null) {
-					data.getCurrentNavigator().navigateTo(JournalView.NAME);
+				if (data.getCurrentNavigator() != null) {
+					data.getCurrentNavigator().navigateTo(data.getCurrentViewName());
```

Comment:
Why did you change this? Doesn't this redirect to the journal view again (when reloading)?

Reply:
If you change the patient you will be on the same view as before but just with the data of the new patient. I think this is more user friendly. E.g.: You want to update medications of different patients you will stay on the medication view if you change the patient.